packages/front-end/components/Experiment/PValueColumn.tsx

```diff
@@ -0,0 +1,127 @@
+import clsx from "clsx";
+import { FC } from "react";
+import { SnapshotMetric } from "back-end/types/experiment-snapshot";
+import { MetricInterface } from "back-end/types/metric";
+import { ExperimentStatus } from "back-end/types/experiment";
+import {
+  hasEnoughData,
+  isBelowMinChange,
+  isSuspiciousUplift,
+} from "@/services/experiments";
+import { useOrganizationMetricDefaults } from "@/hooks/useOrganizationMetricDefaults";
+import usePValueThreshold from "@/hooks/usePValueThreshold";
+import Tooltip from "../Tooltip/Tooltip";
+import NotEnoughData from "./NotEnoughData";
+
+const PValueColumn: FC<{
+  metric: MetricInterface;
+  status: ExperimentStatus;
+  isLatestPhase: boolean;
+  startDate: string;
+  snapshotDate: Date;
+  baseline: SnapshotMetric;
+  stats: SnapshotMetric;
+}> = ({
+  metric,
+  status,
+  isLatestPhase,
+  startDate,
+  snapshotDate,
+  baseline,
+  stats,
+}) => {
+  const {
+    getMinSampleSizeForMetric,
+    metricDefaults,
+  } = useOrganizationMetricDefaults();
+  const pValueThreshold = usePValueThreshold();
+  const minSampleSize = getMinSampleSizeForMetric(metric);
+  const suspiciousChange = isSuspiciousUplift(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
+  const belowMinChange = isBelowMinChange(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
+  const enoughData = hasEnoughData(baseline, stats, metric, metricDefaults);
+  const shouldHighlight =
```

Comment:
nit: Identical to logic in CTWColumn, IDK if worth refactoring.

Reply:
Good call, addressed